src/funcade/jwks.clj

```diff
@@ -2,23 +2,38 @@
   (:require [jsonista.core :as json]
             [buddy.core.keys :as bk]
             [org.httpkit.client :as http]
-            [jsonista.core :as j]
             [funcade.tools :as t]))
 
 (defonce ^:private
-  keyset (atom {}))
+  keyset (atom nil))
+
+(defonce ^:private
+  uri (atom nil))
```

Comment:
why do we need to keep a changeable state for the URI?
`uri` should come from config, as a function argument

if there is no way we can keep it clean: function arg, we should keep it in "`keyset`" atom: `{:keys ... :uri ...}`